### PR TITLE
NXOS - Add received_prefixes in return value and multiple AF

### PR DESCRIPTION
**Pull request by FloLaco**
Hello,

I've improved (I hope so) the `get_bgp_neighbors()` method : 
- Change the `received_prefixes` from `-1` to the real value
- Add support of multiple address family (ipv4, ipv6, vpn4, vpnv6, l2vpn)

Florian

### CODE REVIEW

File: napalm/nxos/nxos.py [hunk 1]
```diff
@@ -982,7 +1003,7 @@ def get_ntp_stats(self):
 
     def get_interfaces_ip(self):
         interfaces_ip = {}
-        ipv4_command = 'show ip interface'
+        ipv4_command = 'show ip interface vrf all'
```
Comment (ktbyers) [inline napalm/nxos/nxos.py]: Why is this changing? That probably should be in a separate change if it needs to change?

File: napalm/nxos/nxos.py [hunk 2]
```diff
@@ -1032,7 +1053,7 @@ def get_interfaces_ip(self):
                     'prefix_length': secondary_address_prefix
                 })
 
-        ipv6_command = 'show ipv6 interface'
+        ipv6_command = 'show ipv6 interface vrf all'
```
Comment (ktbyers) [inline napalm/nxos/nxos.py]: Why is this changing? That probably should be in a separate change if it needs to change?

File: napalm/nxos/nxos.py (hunk 1 above)
Comment (FloLaco) [reply napalm/nxos/nxos.py]: I was adding the vrf support at that time, and yes it's not related to the bgp_neighbors

File: napalm/nxos/nxos.py (hunk 2 above)
Comment (FloLaco) [reply napalm/nxos/nxos.py]: I was adding the vrf support at that time, and yes it's not related to the bgp_neighbors

**Comment by ktbyers** [comment]
@FloLaco Okay, nice...can you back at those changes for the get_interfaces_ip commands?

I think this PR is ready to go besides that.

**Comment by FloLaco** [comment]
@ktbyers sure I will

**Comment by FloLaco** [comment]
@ktbyers it's done. Sorry for the multiple push, I've forgotten the mocked data file

**Pull request merged**